arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Either.kt

```diff
@@ -1944,6 +1951,13 @@ public sealed class Either<out A, out B> {
   )
   public fun void(): Either<A, Unit> =
     map { }
+
+  @Deprecated(
+    "Facilitates the migration from Validated to Either, you can simply remove this method call.",
+    ReplaceWith("this")
+  )
+  public fun toEither(): Either<A, B> =
```

Comment:
Maybe mark is as `inline`, just in case somebody wants to keep it?

Reply:
I'm not sure what you meant 🤔 What would be the purpose of making it `inline`?

Reply:
So that if people leave it, they have no decrease in performance

Reply:
Inline is good 👍

_Sent from my 2201117TG using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Reply:
> So that if people leave it, they have no decrease in performance
They cannot leave it, it's deprecated 😭 😂 

Haha, sounds good 👍

Reply:
```suggestion
  public inline fun toEither(): Either<A, B> =
```